package.json

```diff
@@ -40,21 +38,26 @@
   },
   "homepage": "https://github.com/libp2p/js-libp2p-websockets#readme",
   "dependencies": {
+    "abortable-iterator": "^2.1.0",
     "class-is": "^1.1.0",
     "debug": "^4.1.1",
-    "interface-connection": "~0.3.3",
-    "mafmt": "^6.0.7",
+    "err-code": "^2.0.0",
+    "it-ws": "vasco-santos/it-ws#feat/add-properties-and-functions-to-client-and-server",
```

Comment:
@jacobheun what is your favorite procedure here? Create a tag in my repo's branch?

Reply:
Yeah, tags for dependencies that should eventually get updated. It avoids the volatility of a branch. It's not ideal, but publishing another npm module is overkill imo.

Reply:
ok, updated!